Core/clim-core/graph-formatting.lisp

```diff
@@ -491,8 +491,10 @@ Assumes that GENERATE-GRAPH-NODES has generated only nodes up to the cutoff-dept
   ((stream)
    (arc-drawer)
    (arc-drawing-options)
-   (from-node :initarg :from-node)
-   (to-node :initarg :to-node)))
+   (from-node :initarg :from-node
+              :reader from-node)
+   (to-node :initarg :to-node
+            :reader to-node)))
```

Comment:
please put reader arguments in the same line (this style is mixed across the codebase, but this line is short enough to make it reasonable).